packages/graphback/src/ContextCreator.ts

```diff
@@ -1,26 +1,55 @@
 import { parse, visit } from 'graphql';
-import { Config, Type } from './ContextTypes'
+import { Config, INTERFACE_TYPE_DEFINITION, InterfaceType, Type } from './ContextTypes'
 import { applyGeneratorDirectives } from './directives';
 import { inputTypeVisitor } from './InputTypeVisitor';
 
 /**
- * create input context to be reused for 
- * schema, resolvers generation 
+ * create input context to be reused for
+ * schema, resolvers generation
  * and database creation
  */
-
 export const createInputContext = (schemaText: string, defaultConfig: Config): Type[] => {
   const schema = applyGeneratorDirectives(schemaText)
   try {
     const astNode = parse(schema)
 
-    return visit(astNode, { leave: inputTypeVisitor }).definitions.map((t: Type) => {
+    const schemaDef = visit(astNode, { leave: inputTypeVisitor });
+
+    return schemaDef.definitions.map((t: Type) => {
+
```

Comment:
I would extract that to separate function for redability.

It is that we have 2 steps for building this definitions: 

- Collect actual interfaces in visitors
- Map interface usage in types

It will be cool to have it in single place (visitor itself), but this code is ok as well. We just need to make it clear for future contributors that this is connected to usage of interface